better_bookmarks/js/utils.mjs

```diff
@@ -6,6 +6,7 @@
  *
  */
 
+/* 
```

Comment:
I updated this file, so get the most recent, which you can find from #23, update it, and push it to the branch that you made this pull request from; it will get automatically updated and included in your pull request.

Reply:
I'm just going to mark this one as resolved, and when this file is verified as working, it'll just overwrite this version.